streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -354,11 +355,27 @@ void handleLostAll() {
         }
     }
 
+    /**
+     * @return Map from task id to its total offset summed across all state stores
+     */
+    public Map<TaskId, Long> getTaskOffsetSums() {
+        final Map<TaskId, Long> taskOffsetSums = new HashMap<>();
+
+        for (final TaskId id : tasksOnLocalStorage()) {
+            if (isRunning(id)) {
+                taskOffsetSums.put(id, Task.LATEST_OFFSET);
```

Comment:
I'm just a _tiny_ bit uncomfortable with re-using that sentinel, because the correctness of our logic depends on the active sentinel being less than the standby sentinel, so it _must be_ less than zero. Do we have a reason to believe that `Task.LATEST_OFFSET` would never change to a number that would spoil us here, such as zero?